### PR TITLE
COM-413 add tax number model

### PR DESCRIPTION
**Pull request by sinahashemy**
Adding the new tax number model to the Kontist Client SDK.
**Addresses**: [COM-413](https://kontist.atlassian.net/browse/COM-413)

### CODE REVIEW

File: tests/graphql/taxNumber.spec.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("#fetch", () => {
+    it("should call rawQuery and return all tax numbers details", async () => {
+      // arrange
+      const taxNumbersData = [
+        taxNumberData,
+        { ... taxNumberData, id: "b9ff9d3d-3ab7-452e-a16c-e1611fe443aa" },
```
Comment (dmytroyarmak) [inline tests/graphql/taxNumber.spec.ts:45]: 🙃 It looks strange when there is space after `...`
```suggestion
        { ...taxNumberData, id: "b9ff9d3d-3ab7-452e-a16c-e1611fe443aa" },
```
💭 We probably should enforce some linting rules

**Review by dmytroyarmak — APPROVED** [review]
👍

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "kontist",
-  "version": "0.36.13",
+  "version": "0.36.14",
```
Comment (dmytroyarmak) [inline package.json:3]: ❌ Please don't update version number in the feature PR, because it would cause problems if there is somebody else who is working on the SDK.
Version number should be bumped in a separate PR after this one is merged

**Review by dmytroyarmak — APPROVED** [review]

**Comment by goodfellamilos** [comment]
@sinahashemy We don't add types manually to lib/graphql/schema.ts. See this [slack discussion](https://kontist.slack.com/archives/GACFJPFT6/p1643378795059549).

**Comment by sinahashemy** [comment]
> @sinahashemy We don't add types manually to lib/graphql/schema.ts. See this [slack discussion](https://kontist.slack.com/archives/GACFJPFT6/p1643378795059549).

@goodfellamilos I didn't add any lines to schema.ts and also schema.flow.js, both of them have been extended with graphQL-code-generator. (npm run schema:generate).

**Review by goodfellamilos — APPROVED** [review]
Sorry, my bad, schema looks good 👍  What's the reason for removing `package-lock.json`?
